### DISCUSSION TITLE
Banned Users disable video

### DISCUSSION THREAD

**Discussion by sitehatchery** [owncast/owncast#1437] [Q&A] [answered]
I want a banned user to not only be banned from the chat, but also from seeing the video.

I was looking for a way to identify the banned user through my website. The only way I could think is to push a unique ID relating to the user/session via the API at the beginning of the user's session (they are logged into a page).  

I'd then make an API request  and then get back the user status from Owncast. I'd make the api request every minute or so and if the user is banned, then I would have logic to hide the video and kick them out. 

Is there already this capability or another way of doing it?

**Comment by gabek** [comment, marked as answer]
There is no way to do that currently, and it's not on our roadmap.  Removing people from chat has been the focus.  However there's no reason you couldn't remove the video when chat is removed as well.  https://github.com/owncast/owncast/blob/develop/webroot/js/app.js#L543 (this link to code will probably change over time as code changes, but it's accurate now)

**Comment by sitehatchery** [comment]
Thanks!
